js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeMiddlePanelNoPartitions.tsx

```diff
@@ -57,14 +57,28 @@ export const AutomaterializeMiddlePanelNoPartitions = ({
         border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
         flex={{alignItems: 'center', justifyContent: 'space-between'}}
       >
-        <Subheading>Result</Subheading>
-        <div>{headerRight()}</div>
+        <Subheading>Evaluation</Subheading>
       </Box>
       <ConditionsNoPartitions
         conditionResults={conditionResults}
         maxMaterializationsPerMinute={maxMaterializationsPerMinute}
         parentOutdatedWaitingOnAssetKeys={parentOutdatedWaitingOnAssetKeys}
       />
+      <Box
+        // style={{flex: '0 0 96px'}}
+        border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
+        flex={{alignItems: 'center', justifyContent: 'space-between'}}
+      >
+        <Box
+          style={{flex: '0', backgroundColor: Colors.Gray100}}
+          padding={{horizontal: 32, vertical: 24}}
+        >
+          <Subheading>Result</Subheading>
+        </Box>
```

Comment:
This style seems very unusual to me. Are there other places in the app where we use a padded gray box this way?